src/midifunctor.cpp

```diff
@@ -766,12 +766,12 @@ FunctorCode GenerateMIDIFunctor::VisitNote(const Note *note)
         return FUNCTOR_SIBLINGS;
     }
 
-    const int channel = m_midiChannel;
-    int velocity = MIDI_VELOCITY;
-    if (note->HasVel()) velocity = note->GetVel();
+    const int velocity = (note->HasVel()) ? note->GetVel() : MIDI_VELOCITY;
```

Comment:
Interesting didactic use of `const`: eventually `velocity` should be modified to reflect the note's articulations (accents, where roughly 10 should be added to the velocity and clipped if exceeding 127), prevailing dynamics markings (harder), (and hairpins which are hardest).